test/models/user_test.rb

```diff
@@ -214,5 +214,28 @@ class UserTest < ActiveSupport::TestCase
       @user.reload
       assert @user.disabled_by_fund
     end
+
+    describe 'nightly cleanup task - disable_inactive_users' do
+      before do
+        @disabled_user = create(:user, last_sign_in_at: 10.months.ago)
```

Comment:
Idea: given you have a couple different cases, look into doing this test with a `DataTable`?

Reply:
per discussion in slack, there isn't a fantastic model for this to glom onto in ruby rn, so punting.